plugin.go

```diff
@@ -113,6 +114,11 @@ func (p *Plugin) Exec() error {
 		p.ContainerName = p.Family + "-container"
 	}
 
+	// Fargate doesn't support privileged mode
+	if (p.Compatibilities == "FARGATE") {
+		p.Privileged = false
```

Comment:
Do you mind adding some logging here just to state that we're possibly ignoring the privileged value if we're using Fargate?  I'm sure someone will appreciate it if they accidentally think it should work :)